generator/Makefile

```diff
@@ -0,0 +1,5 @@
+golden:
```

Comment:
nit: this is missing the Copyright boiler plate.

Reply:
ack